src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import missionsReducer, { fetchMissionsFromApi } from './missions/missions';
+import { rocketReducer } from './rockets/rockets';
+
+const reducer = combineReducers({
+  missionsReducer, rocketReducer,
+});
+
+const store = createStore(reducer, applyMiddleware(thunk, logger));
+store.dispatch(fetchMissionsFromApi());
```

Comment:
- [ ] You may consider dispatching fetchMissionsFrom Api in the App component and make this file only responsible for initializing our redux store.

Reply:
Hello @wandji20, since even you yourself said I could consider it, would you mind making this not required but only optional? The code is working the same and it'll just make us take longer to complete the project. Thank you!

Reply:
Concerning the fetchMissionsFromApi request, you have already done the heavy work creating the async action creator (fetchMissionsFromApi). Similar to how you fetch rocket data with getRocketData async action creator in the rockets/List.jsx, you can call a useEffect in the App component and have it dispatch fetchMissionsFromApi. This way you avoid mixing up your logic and keep your codebase easier to understand.